redskyctl/internal/commands/run/view.go

```diff
@@ -0,0 +1,420 @@
+/*
+Copyright 2021 GramLabs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package run
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"text/tabwriter"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/thestormforge/optimize-controller/redskyctl/internal/commands/run/form"
+	"github.com/thestormforge/optimize-controller/redskyctl/internal/commands/run/internal"
+	"github.com/thestormforge/optimize-controller/redskyctl/internal/commands/run/out"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+const (
+	ScenarioTypeStormForger = "StormForge"
+	ScenarioTypeLocust      = "Locust"
+)
+
+var pagerKeyBindings = []out.KeyBinding{
+	{
+		Key:  tea.Key{Type: tea.KeyCtrlX},
+		Desc: "Quit",
+	},
+	{
+		Key:  tea.Key{Type: tea.KeySpace},
+		Desc: "Next page",
+	},
+	{
+		Key:  tea.Key{Type: tea.KeyRunes, Runes: []rune{'b'}},
+		Desc: "Previous page",
+	},
+}
+
+// initializeModel is invoked before the program is started to ensure things
+// are in a valid state prior to starting.
+func (o *Options) initializeModel() {
+	var opts []out.FieldOption
+	if o.Verbose {
+		opts = append(opts, out.VerbosePrompts)
+	}
+
+	o.generatorModel.ScenarioType = out.FormField{
+		Prompt:       "Where do you want to get your load test from?",
+		Instructions: []string{"up/down: select", "pgup: back", "enter: continue"},
```

Comment:
nit/dry, we could probably move some of these instructions to a variable. not a blocker but I could see this being a chore to update in the future.

Reply:
Yeah, furthermore pgup and enter are constant for all of them.